website/server/controllers/api-v3/challenges.js

```diff
@@ -17,9 +20,81 @@ import {
 import * as Tasks from '../../models/task';
 import Bluebird from 'bluebird';
 import csvStringify from '../../libs/csvStringify';
+import {
+  createTasks,
+} from '../../libs/taskManager';
+
+const TASK_KEYS_TO_REMOVE = ['_id', 'completed', 'date', 'dateCompleted', 'history', 'id', 'streak', 'createdAt', 'challenge'];
```

Comment:
I think `'date'` should not be in this list - i.e., it should be cloned into the new challenge's tasks. It's the due date for To-Dos, which the challenge creator is allowed to set.

Reply:
This is copied directly from the old site, so I'm surprised we didn't notice that before, but I'm totally fine changing it!